code/modules/mod/mod_control.dm

```diff
@@ -361,6 +362,7 @@
 			playsound(src, 'sound/machines/scanbuzz.ogg', 25, TRUE, SILENCED_SOUND_EXTRARANGE)
 			return FALSE
 		install(attacking_item, user)
+		SEND_SIGNAL(src, COMSIG_MOD_MODULE_CHANGED, user)
```

Comment:
use different signals, don't send the same signal in different places. Use signals like COMSIG_MOD_MODULE_REMOVED, COMSIG_MOD_MODULE_ADDED

Reply:
will do